modules/nw-sriov-dpdk-example-intel.adoc

```diff
@@ -118,7 +118,7 @@ spec:
 <2> Specify the DPDK image which includes your application and the DPDK library used by application.
 <3> Specify the `IPC_LOCK` capability which is required by the application to allocate hugepage memory inside container.
 <4> Mount a hugepage volume to the DPDK Pod under `/dev/hugepages`. The hugepage volume is backed by the emptyDir volume type with the medium being `Hugepages`.
-<5> (optional) Specify the number of DPDK devices allocated to DPDK Pod. This resource request and limit, if not explicitly specified, will be automatically added by the SR-IOV network resource injector. The SR-IOV network resource injector is an admission controller component managed by the SR-IOV Operator. It is enabled by default and can be disabled by setting `Injector` option to `false` in the default `SriovOperatorConfig` CR.
+<5> Optional: Specify the number of DPDK devices allocated to DPDK Pod. This resource request and limit, if not explicitly specified, will be automatically added by the SR-IOV network resource injector. The SR-IOV network resource injector is an admission controller component managed by the SR-IOV Operator. It is enabled by default and can be disabled by setting `Injector` option to `false` in the default `SriovOperatorConfig` CR.
```

Comment:
@zshi-redhat, is this correct? Thanks!

Reply:
@kalexand-rh  @jboxman these are two different CRs although they look similar.
`SriovNetworkNodeState` is created and managed by Operator, should not be changed by user.
`SriovOperatorConfig` is created by Operator, it can be edited by user. There is only one CR of `SriovOperatorConfig` type in the cluster, named `default`.

Reply:
@jboxman 
please update `Injector` to `enableInjector`